File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/ExperimentReportVO.kt
package com.sdkwork.app

data class ExperimentReportVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val experimentKey: String? = null,
    val reportType: String? = null,
    val reportData: Map<String, Any>? = null,
    val generatedAt: String? = null,
    val version: String? = null
)

File: sdkwork-app-sdk-typescript/src/types/experiment-report-vo.ts
/** 实验报告响应 */
export interface ExperimentReportVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 实验标识 */
  experimentKey?: string;
  /** 报告类型 */
  reportType?: string;
  /** 报告数据 */
  reportData?: Record<string, unknown>;
  /** 报告生成时间 */
  generatedAt?: string;
  /** 报告版本 */
  version?: string;
}
